pkg/service/one2onerestore/worker.go

```diff
@@ -218,6 +218,11 @@ func (w *worker) prepareHostWorkload(ctx context.Context, manifests []*backupspe
 			return errors.Wrapf(err, "node %s safe describe method", h.Addr)
 		}
 		hw.host.SafeDescribeMethod = method
+		supports, err := nodeInfo.SupportsSkipCleanupAndSkipReshape()
+		if err != nil {
+			return errors.Wrapf(err, "node %s supports skip_cleanup and skip_reshape", h.Addr)
+		}
+		hw.host.SkipCleanupAndSkipReshape = supports
```

Comment:
nit: I would just store NodeInfo alongside host instead of populating host with simple method calls here. NodeInfo could also be taken from the node config service later on.
On the other hand, since we are already doing it for safe describe schema with internals, it's fine to go along with this pattern.